Add changelog for v1.6.1 (#1022)
auth0/terraform-provider-auth0: CHANGELOG.md

## v1.6.1

BUG FIXES:

> - `resource/auth0_client`: Resolve an issue where default_organization plan failed due to ConflictsWith/RequiredWith clauses in schema ([#1021](https://github.com/auth0/terraform-provider-auth0/pull/1021))

Reviewer: ```suggestion
- `resource/auth0_client`: Fixed an issue where the `default_organization` plan failed due to `ConflictsWith` and `RequiredWith` clauses in the schema ([#1021](https://github.com/auth0/terraform-provider-auth0/pull/1021))
```
